addons/website_sale/models/product.py

```diff
@@ -191,14 +191,15 @@ def _website_price(self):
         qty = self._context.get('quantity', 1.0)
         partner = self.env.user.partner_id
         pricelist = self.env['website'].get_current_website().get_current_pricelist()
+        company_id = self.env['website'].get_current_website().company_id
 
         context = dict(self._context, pricelist=pricelist.id, partner=partner)
         self2 = self.with_context(context) if self._context != context else self
 
         ret = self.env.user.has_group('sale.group_show_price_subtotal') and 'total_excluded' or 'total_included'
 
         for p, p2 in zip(self, self2):
-            taxes = partner.property_account_position_id.map_tax(p.taxes_id)
+            taxes = partner.property_account_position_id.map_tax(p.taxes_id).filtered(lambda x: x.company_id == company_id)
```

Comment:
@pedrobaeza 
what do you mean by 'put filter before'?
It is map_tax that returns the list of taxes, before the map tax, you just have one 'account.fiscal.position'